scripts/release.ts

```diff
@@ -40,34 +45,41 @@ export async function release(rootDir: string, args: string[]) {
   }
 }
 
-async function prepareRelease(opts: BuildOptions, args: string[], releaseDataPath: string) {
+/**
+ * Prepares a release of Stencil
+ * @param opts build options containing the metadata needed to release a new version of Stencil
+ * @param args stringified arguments used to influence the release steps that are taken
+ * @param releaseDataPath the fully qualified path of `release-data.json` to write to disk during this step
+ */
+async function prepareRelease(opts: BuildOptions, args: ReadonlyArray<string>, releaseDataPath: string): Promise<void> {
   const pkg = opts.packageJson;
   const oldVersion = opts.packageJson.version;
   console.log(
     `\nPrepare to publish ${opts.vermoji}  ${color.bold.magenta(pkg.name)} ${color.dim(`(currently ${oldVersion})`)}\n`
   );
 
+  const NON_SERVER_INCREMENTS: ReadonlyArray<{ name: string; value: string }> = [
+    {
+      name: 'Dry Run',
+      value: getNewVersion(oldVersion, 'patch') + '-dryrun',
+    },
+    {
+      name: 'Other (specify)',
+      value: null,
+    },
+  ];
+
   const prompts = [
     {
       type: 'list',
       name: 'version',
       message: 'Select semver increment or specify new version',
-      pageSize: SEMVER_INCREMENTS.length + 2,
+      pageSize: SEMVER_INCREMENTS.length + NON_SERVER_INCREMENTS.length,
       choices: SEMVER_INCREMENTS.map((inc) => ({
         name: `${inc}   ${prettyVersionDiff(oldVersion, inc)}`,
         value: inc,
-      })).concat([
-        new inquirer.Separator() as any,
-        {
-          name: 'Dry Run',
-          value: getNewVersion(oldVersion, 'patch') + '-dryrun',
-        },
-        {
-          name: 'Other (specify)',
-          value: null,
-        },
-      ]),
-      filter: (input: any) => (isValidVersionInput(input) ? getNewVersion(oldVersion, input) : input),
```

Comment:
This felt a little iffy to me - I really didn't like that 'magic number 2' above, and this is how I though it'd best be refactored to remove that. LMK if you'd prefer I revert this

Reply:
lgtm